wolfssl/wolfcrypt/asn.h

```diff
@@ -2733,6 +2739,11 @@ struct OcspEntry
     WC_BITFIELD used:1;                   /* entry used                */
 };
 
+enum responderIdType {
+    OCSP_RESPONDER_ID_NAME = 0,
+    OCSP_RESPONDER_ID_KEY  = 1,
+    OCSP_RESPONDER_ID_INVALID = 2
+};
```

Comment:
Since `resp->responderIdType` is `0` by default at init (I'm guessing) it would be better to make `0` the invalid value. This way there is no ambiguity if it has been set or not.